gooseai/generation/generation.pb.go

```diff
@@ -951,7 +996,7 @@ type PromptParameters struct {
 	unknownFields protoimpl.UnknownFields
 
 	Init   *bool    `protobuf:"varint,1,opt,name=init,proto3,oneof" json:"init,omitempty"`
-	Weight *float32 `protobuf:"fixed32,2,opt,name=weight,proto3,oneof" json:"weight,omitempty"`
+	Weight *float64 `protobuf:"fixed64,2,opt,name=weight,proto3,oneof" json:"weight,omitempty"`
```

Comment:
I find it interesting that this is changing from float32 to float64. (I'm not saying it's a bad thing - but as I'm reviewing this, I'm curious)

Reply:
The reason is python float is a C-style 64-bit double, so I would rather to use float64 in the protobuf to avoid python lost float precision.